test/classes/initializers/records/generics/fullySpecifiedAssignment/varArgDefaultValue.chpl

```diff
@@ -9,7 +9,7 @@ record Foo {
   }
 }
 
-var foo1: Foo(bool) = new Foo(true);
+var foo1: Foo(int) = new Foo(true);
```

Comment:
I'm not sure if I think this change in behavior is correct, but I'm willing to be convinced :)

Reply:
new Foo(true) in this test creates a Foo(int) -- the test says so itself below. So isn't it unreasonable to try to store that into a Foo(true) ?

Reply:
Fair point